services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> None:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+        hall_name: None,
+        hall_rows: None,
+        hall_seats_in_row: None
+) -> None:

```

Comment:
The function `create_cinema_hall` should not return `None`. It should return the created `CinemaHall` instance. Update the return type to `CinemaHall`.